### PR TITLE
CONSOLE-3185: Improve information available for Pending or Failed plugins so user can better troubleshoot plugins

### PR DESCRIPTION
**Pull request by jcaianirh**
Addresses: https://issues.redhat.com/browse/CONSOLE-3185

Screen Shot 2022-10-24 at 5 18 42 PM
Screen Shot 2022-10-24 at 5 17 17 PM
Screen Shot 2022-10-24 at 5 15 15 PM

cc: @megan-hall

### CODE REVIEW

**Comment by rhamilto** [comment]
Nice work, @jcaianirh.  Will let @jhadvig weigh in before tagging.

**Comment by megan-hall** [comment]
LGTM!

**Comment by jcaianirh** [comment]
QE Approver:
/assign @yapei

Docs Approver:
/assign @opayne1

PX Approver:
/assign @RickJWagner

Console Approver:
/assign @jhadvig

**Review by jhadvig — APPROVED** [review]
/lgtm
/approve

File: frontend/packages/console-app/src/components/console-operator/ConsoleOperatorConfig.tsx [hunk 1]
```diff
@@ -121,6 +142,15 @@ const ConsolePluginsList: React.FC<ConsolePluginsListType> = ({ obj }) => {
             },
             item.version || placeholder,
             item.description || placeholder,
+            item.status
+              ? {
+                  title: (
+                    <Status status={item.status} title={item.status}>
+                      {item.errorMessage}
```
[the remark this answers is not in the document]
Comment (jcaianirh) [reply frontend/packages/console-app/src/components/console-operator/ConsoleOperatorConfig.tsx]: Also added error cause into the popover message
Screen Shot 2022-10-27 at 12 19 27 PM

**Review by rhamilto — APPROVED** [review]
/lgtm

**Comment by yapei** [comment]
`Console plugins` page has 'Status' column which will show detailed errorMessage and errorCause for 'Failed' plugins
/label qe-approved

**Pull request merged**